ui/shared/components/panel/view-pedigree-image/PedigreeImageButtons.jsx

```diff
@@ -33,25 +33,40 @@ class BaseEditPedigreeImageButton extends React.PureComponent {
       this.setState({ error: `Error: ${xhr.statusText} (${xhr.status})` })
   }
 
+  confirmNoPhi = () => {
+    this.setState({ confirmedNoPhi: true })
+  }
+
   render() {
     const { family } = this.props
-    const { error } = this.state
+    const { error, confirmedNoPhi } = this.state
     return (
       <Modal
         title={`Upload Pedigree for Family ${family.familyId}`}
         modalName={this.modalId}
         trigger={<ButtonLink content="Upload New Image" icon="upload" labelPosition="right" />}
       >
-        <React.Suspense fallback={<Loader />}>
-          <XHRUploaderWithEvents
-            onUploadFinished={this.onFinished}
-            url={`/api/family/${family.familyGuid}/update_pedigree_image`}
-            clearTimeOut={0}
-            auto
-            maxFiles={1}
-            dropzoneLabel="Drag and drop or click to upload pedigree image"
-          />
-        </React.Suspense>
+        {confirmedNoPhi ? (
+          <React.Suspense fallback={<Loader />}>
+            <XHRUploaderWithEvents
+              onUploadFinished={this.onFinished}
+              url={`/api/family/${family.familyGuid}/update_pedigree_image`}
+              clearTimeOut={0}
+              auto
+              maxFiles={1}
+              dropzoneLabel="Drag and drop or click to upload pedigree image"
+            />
+          </React.Suspense>
+        ) : (
+          <Segment basic compact textAlign="center" size="large">
+            <i>seqr </i>
+            is not a HIPAA-compliant platform. By proceeding, I affirm that this image does not contain any
+            protected health information (PHI), either in the image itself or in the image metadata.
+            <br />
+            <br />
+            <Button primary floated="right" content="Continue" onClick={this.confirmNoPhi} />
```

Comment:
Do we also need a cancel button?

Reply:
no theres already an x button on the modal